src/lib/ext/multi_class.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-throw-literal */
+import * as regex from "../regex.js";
+
+const MultiClassError = new Error();
+
+/**
+ *
+ * @param {CompiledMode} mode
+ */
+export function MultiClass(mode) {
+  if (!Array.isArray(mode.begin)) return;
```

Comment:
```suggestion
  if (!Array.isArray(mode.begin)) {
    mode.isMultiClass = false;
    return;
  }
```

`isMultiClass` should be set to false since according to the type definition, it's a boolean that will exist in a CompiledMode.

Reply:
I made the type optional instead.